adafruit_motor/motor.py

```diff
@@ -23,11 +23,25 @@
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_Motor.git"
 
+from micropython import const
+
+FAST_DECAY = const(0)  # Recirculation current fast decay mode (coasting)
+SLOW_DECAY = const(1)  # Recirculation current slow decay mode (braking)
```

Comment:
No need for const because it only does something on private (aka starts with _) variables. Also, changes the comment so they'll appear in ReadTheDocs.
```suggestion
FAST_DECAY = 0
"""Recirculation current fast decay mode (coasting)"""

SLOW_DECAY = 1
"""Recirculation current slow decay mode (braking)"""
```